### PR TITLE
fixed some comment issue

### PR DESCRIPTION
**Pull request by undefinedor**
| Q             | A
| ------------- | ---
| Is bugfix?    |  no
| New feature?  | no
| Breaks BC?    |  no
| Tests pass?   |  no

just some comment issue

### CODE REVIEW

**Comment by samdark** [comment]
Thank you!

File: framework/base/Module.php [hunk 1]
```diff
@@ -356,9 +356,7 @@ protected function defaultVersion()
      * Defines path aliases.
      * This method calls [[Yii::setAlias()]] to register the path aliases.
      * This method is provided so that you can define path aliases when configuring a module.
-     * @property array list of path aliases to be defined. The array keys are alias names
-     * (must start with `@`) and the array values are the corresponding paths or aliases.
-     * See [[setAliases()]] for an example.
```
Comment (cebe) [inline framework/base/Module.php]: why did you remove this? It totally belongs here...

**Comment by cebe** [comment]
reverted: c1c11d0, @yiisoft/core-developers FYI.

File: framework/base/Module.php (hunk 1 above)
Comment (undefinedor) [inline framework/base/Module.php]: it's duplicated.
```
/**
     * Defines path aliases.
     * This method calls [[Yii::setAlias()]] to register the path aliases.
     * This method is provided so that you can define path aliases when configuring a module.
     * @property array list of path aliases to be defined. The array keys are alias names
     * (must start with `@`) and the array values are the corresponding paths or aliases.
     * See [[setAliases()]] for an example.
     * @param array $aliases list of path aliases to be defined. The array keys are alias names
     * (must start with `@`) and the array values are the corresponding paths or aliases.
     * For example,
     *
     * ```php
     * [
     *     '@models' => '@app/models', // an existing alias
     *     '@backend' => __DIR__ . '/../backend',  // a directory
     * ]
     * ```
     */
```
 Don't you think it's duplicated?

Comment (cebe) [inline framework/base/Module.php]: No, see my comment in the commit that reverts it. This annotation is to generate the `@property ` annotation in class PHPdoc for this setter.

Comment (undefinedor) [inline framework/base/Module.php]: I see, it's used to generate annotation.  You are right .
